rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -1007,4 +1092,18 @@ public void addEmbeddedComponentOwnerToComponent(HalResource halResource, Compon
             }
         }
     }
+
+    public void addEmbeddedSubcribeToHalResourceComponent(HalResource halResource, Component sw360Component) {
```

Comment:
This method has nothing to do with this PR, right? you just added for future use?

Reply:
Yes, because PR(https://github.com/eclipse-sw360/sw360/pull/1983) was merged before, to avoid duplicate PR I added it to this same PR.